java/java_test.go

```diff
@@ -1,27 +1,52 @@
-package samples_test
+package java_test
 
 import (
+	"flag"
 	"fmt"
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
+	"time"
 
 	"github.com/paketo-buildpacks/occam"
+	"github.com/paketo-buildpacks/samples/tests"
 	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
 
 	. "github.com/onsi/gomega"
 	. "github.com/paketo-buildpacks/occam/matchers"
```

Comment:
How do you all typically group import statements in the other paketo repos. In the java buildpack repos (and cnb repos) we typically have three groups:
1. stdlib (e.g. `os`)
2. external (e.g. `github.com/paketo-buildpacks/occam`)
3. internal (e.g. `github.com/paketo-buildpacks/samples/tests`)

From this PR and other paketo buildpack repos am guessing you typically combine groups 2&3 but separate dot imports? But we don't seem to be [totally consistent about it](https://github.com/paketo-buildpacks/occam/blob/main/buildpack_store_test.go#L9-L12). No need to change anything here but maybe this is something we should add to the style guide and make consistent accross the project?

Reply:
Sounds reasonable. Can you open an issue [on the community repo](https://github.com/paketo-buildpacks/community/issues) to propose a change to the style guide?

Reply:
https://github.com/paketo-buildpacks/community/issues/17